lib/blpapi-wrapper.ts

```diff
@@ -71,15 +71,25 @@ var EVENT_TYPE = {
   , PARTIAL_RESPONSE: 'PARTIAL_RESPONSE'
 };
 
+// Map of HTTP request names to the BLAPI request and response names.
+var NAME_MAP: { [s: string]: { reqName: string; resName: string; } } = {
+    'FieldInfo':        { reqName: 'FieldInfoRequest',
+                          resName: 'fieldResponse' },
+    'FieldSearch':      { reqName: 'FieldInfoRequest',
+                          resName: 'fieldResponse' },
+    'InstrumentLookup': { reqName: 'instrumentListRequest',
+                          resName: 'InstrumentListResponse' },
+    'CurveLookup':      { reqName: 'curveListRequest',
+                          resName: 'CurveListResponse' },
+    'GovernmentLookup': { reqName: 'govtListRequest',
+                          resName: 'GovtListResponse' }
+};
+
```

Comment:
Seems to missing `HistoricalData` & `ReferenceData` in this list(The other two I am aware of). I am not sure if there's a complete list of request names in the api doc though.